src/loader.test.ts

```diff
@@ -63,7 +64,7 @@ describe('load', () => {
     expect(fetchMock.lastUrl?.host).toStrictEqual('api-prefab-cloud.global.ssl.fastly.net');
     expect(fetchMock.lastRequestOptions?.headers).toStrictEqual({
       Authorization: 'Basic dTphcGlLZXk=',
-      'X-PrefabCloud-Client-Version': 'prefab-cloud-js0.0.1',
+      'X-PrefabCloud-Client-Version': `prefab-cloud-js${version}`,
```

Comment:
This fix is included on every open PR 😅 It ensures the version matches the current version in the package.json